deploy/efs-csi/05-scc.yaml

```diff
@@ -0,0 +1,39 @@
+# A clone of the `anyuid` scc allowing dedicated-admins to create pods
```

Comment:
Is this a literal clone?  If yes, what keeps us from using `anyuid` as it is?

Reply:
I'm glad you asked.

I couldn't get a patch (to add `dedicated-admins` to `anyuid`) to work idempotently. I tried all three `patchType`s. It would either replace the `groups` list completely, or add `dedicated-admins` every time it ran (so the list kept growing), or fail at the REST request for a bad header (`Accept` or `Content-Type`, can't remember which). It's possible there's a bug in whatever implementation of strategic merge we're using.

Reply:
We shouldn't be modifying default SCCs to add dedicated-admins to them.. it'll break upgrades in 4.3 and will be impossible in 4.4+ (ref: https://bugzilla.redhat.com/show_bug.cgi?id=1821905)

Cloning the SCC is the right approach, if the SCC is required.

Reply:
I know very little about SCCs but the docs show you can add SCC in a role.  We should try not to clone the SCC as it may change in future OCP releases.  You cannot modify it because something on cluster is managing it, either CVO or another operator. 

https://docs.openshift.com/container-platform/4.3/authentication/managing-security-context-constraints.html#role-based-access-to-ssc_configuring-internal-oauth

Reply:
It doesn't really matter if the `anyuid` SCC diverges from this one in the future. The important thing is that this one allows creation of a root pod, which it will continue to do regardless. This approach doesn't stop `dedicated-admins` being added to other SCCs (including `anyuid` for that matter). And we don't *want* this one to get modified in situ, which is another advantage of having it separate from `anyuid` (patch problems aside).

Reading what you linked, it does look like we could create a Role and (Cluster)RoleBinding to give access to the `anyuid` SCC. But I don't really see what difference it would make.

Reply:
Update: we don't need the SCC after all, if we use access points to set up the PV. I removed it in the latest commit.